src/components/forms/validators/__specs__/validateEmailField.spec.js

```diff
@@ -0,0 +1,31 @@
+import strings from '../strings'
+import validateEmailField from '../validateEmailField'
+
+describe('src | components | forms | validators | validateEmailField', () => {
+  it('expect to return default error - not an email', () => {
```

Comment:
```suggestion
  it('expect to return default error - not a string', () => {
```
Vu qu'en dessous tu testes `''` et `'      '` qui ne sont clairement pas des emails non plus

Reply:
D'ailleurs, est-ce que ces cas de tests sont plausibles ?
ie un utilisateur peut-il réussir à fourrer un booléen ou une erreur dans notre input ? :monocle_face:

Reply:
OK